gradle-plugin/src/main/groovy/ch/silviowangler/gradle/restapi/PluginTypes.java

```diff
@@ -31,9 +31,6 @@
 public enum PluginTypes {
 
 	RESTAPI_IDTYPE(ClassName.get("ch.silviowangler.rest.types", "IdType")),
-	RESTAPI_FILTERMODEL(ClassName.get("ch.silviowangler.rest.model", "FilterModel")),
```

Comment:
The removal of these constants is not part of the PR. Please revert the changes on this file